### DISCUSSION TITLE
[Feature] Separate image/video fg/bg upload settings

### DISCUSSION THREAD

**Discussion by Fmstrat** [immich-app/immich#10639] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The current upload upload implementation has a few gaps when compared to other solutions like Nextcloud.

**Don't treat Image and videos the same**
If I go on vacation, or out for the day, and take 20 pictures and a 1GB video, I have to make a conscious decision beforehand to "only upload on WiFi" because of the video size taking up so much bandwidth.

Nextcloud solves this by having separate image and video upload settings. I.E. the user can have images upload always, but video on WiFi only.

**Add WiFi/charging options to foreground mode**
In a similar vain, with these settings missing from foreground mode, if you use foreground mode to save bandwidth, you can't open the app when out amd about or you risk a large upload over your data plan.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3341

**Comment by Fmstrat** [comment]
Ahhh, missed that. Sorry.
